pkg/apis/tenancy/v1alpha1/types.go

```diff
@@ -221,6 +221,27 @@ type ClusterWorkspaceTypeSpec struct {
 	//
 	// +optional
 	LimitAllowedParents *ClusterWorkspaceTypeSelector `json:"limitAllowedParents,omitempty"`
+
+	// defaultAPIBindings are the APIs to bind during initialization of workspaces created from this type.
+	// The APIBinding names will be generated dynamically.
+	//
+	// +optional
+	DefaultAPIBindings []APIExportReference `json:"defaultAPIBindings,omitempty"`
+}
+
+// APIExportReference provides the fields necessary to resolve an APIExport.
+type APIExportReference struct {
+	// path is the fully-qualified path to the workspace containing the APIExport.
+	//
+	// +required
+	// +kubebuilder:Required
+	Path string `json:"path"`
+
+	// name is the name of the APIExport.
+	//
+	// +required
+	// +kubebuilder:Required
+	Name string `json:"name"`
```

Comment:
pattern

Reply:
i'm adding minlength=1, which is consistent w/what's in apibinding